parsec/core/gui/custom_dialogs.py

```diff
@@ -636,7 +646,11 @@ def show_info_copy_link(
     parent: QWidget, title: str, message: str, button_text: str, url: str
 ) -> None:
     widget = InfoCopyLinkWidget(message, url, button_text)
-    dialog = GreyedDialog(widget, title=title, parent=parent)
+    dialog = GreyedDialog(
+        cast(InfoLinkWidget, widget),  # Upcast to avoid type error with mypy
```

Comment:
Why `mypy` generate an error here ?

Reply:
For some reasons mypy refuse to convert a `InfoCopyLinkWidget` to a `InfoLinkWidget` despite `InfoCopyLinkWidget` is derived from `InfoLinkWidget`.

Reply:
That's strange :thinking: